xhr.bs

```diff
@@ -960,14 +962,13 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
    <li><p>If <var>processedResponse</var> is false, then set <a>this</a>'s <a>timed out flag</a> and
    <a for="fetch controller">terminate</a> <a>this</a>'s <a for=XMLHttpRequest>fetch controller</a>.
 
+   <li><p>Call <a>this</a>'s <a for=XMLHttpRequest>fetch controller</a>'s
+   <a for="fetch controller">report timing</a> given the <a>current global object</a>.
```

Comment:
This would be a bit more conventional as "Report timing for this's fetch controller given the current global object".

Reply:
I'll push a fixup commit.